requirements/base.txt

```diff
@@ -65,13 +65,13 @@ jinja2==3.1.2
     # via code-annotations
 lazy==1.5
     # via bok-choy
-lazy-object-proxy==1.8.0
+lazy-object-proxy==1.9.0
     # via astroid
 markupsafe==2.1.1
     # via jinja2
 mccabe==0.7.0
     # via pylint
-mock==4.0.3
+mock==5.0.0
```

Comment:
Some major changes in the new version. May affect behaviour of mocked tests but will be found out later on https://github.com/testing-cabal/mock/blob/master/CHANGELOG.rst#500.